### DISCUSSION TITLE
Query Suggestion activated but doesn't suggest anything

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2861] [Q&A]
Hi all,

Would you mind telling me what this query suggestion in search box stands for? I type something like J etc. but nothing appears until I write J* or Jazz. I also considered the page for custom suggestion but I don't get it. 

![1E3BAA8D-12E5-4CE0-8AF3-180F6BEF830B_4_5005_c](https://user-images.githubusercontent.com/124573411/225951139-fa98950d-4c09-4342-ab6b-d99306488991.jpeg)

**Comment by kasperbolarsen** [comment]
See review https://github.com/microsoft-search/pnp-modern-search/discussions/2197
And us know if it answered your question.

**Reply by sayuri8** [reply]
@kasperbolarsen Thanks for your response. I tried to reach this site which was indicated: https://tenant-admin.sharepoint.com/_layouts/15/searchadmin/querysuggestionsettings.aspx?level=tenant 

But it doesn't work. I am collection admin. Is this enough to make the adjustment?

**Reply by kasperbolarsen** [reply]
I am pretty sure you will have to be a SharePoint admin.
Don't you have access to the Developer Program: https://developer.microsoft.com/en-us/microsoft-365/dev-program

This will provide a tenant for dev/test purposes
